### ISSUE TITLE
Implement KHR_lights

### ISSUE THREAD

**Issue by donmccurdy** [KhronosGroup/glTF-Blender-Exporter#178]
The draft specification (https://github.com/KhronosGroup/glTF/pull/1223) is now stable enough to consider implementing here.

**Comment by donmccurdy** [comment]
Fixed by #223.

**Comment by IARI** [comment]
How can I export a colored light?

Sorry to bother this issue again - but the code clearly seems to suggest that a color value from Blender is exported. However in the output gltf all lights I export will have the property
```
    "color" : [
        1.0,
        1.0,
        1.0
    ],
```
while in Blender I simply added the lights and then changed its color:
![image](https://user-images.githubusercontent.com/1053417/42163241-0c5e1728-7e02-11e8-917e-831f7c30c9d4.png)

I'm using cycles, otherwise I cannot use the gltf-metalness node for material export.
